### PR TITLE
chore: Expose REPLACED on CAS side

### PR DESCRIPTION
**Pull request by ukstv**

### CODE REVIEW

File: src/services/request-presentation-service.ts [hunk 1]
```diff
@@ -98,7 +98,6 @@ export class RequestPresentationService {
       case RequestStatus.PROCESSING:
       case RequestStatus.FAILED:
       case RequestStatus.READY:
-        return this.notCompleted(request, status)
```
Comment (stbrody) [inline src/services/request-presentation-service.ts:101]: I'm confused by this change.  Wouldn't this make all of PENDING, PROCESSING, READY, and REPLACED all come back to Ceramic as FAILED?  I would have thought that the change to make would be removing the logic that overrides the `status` field on line 106

Comment (ukstv) [reply src/services/request-presentation-service.ts:101]: Agree. Rebase went wrong.

Comment (stbrody) [reply src/services/request-presentation-service.ts:101]: seems like tests should have caught that?  Can we add some test coverage that would have prevented us from accidentally putting in a regression like that if I hadn't noticed the issue in code review?

Comment (ukstv) [reply src/services/request-presentation-service.ts:101]: They did catch that :) ceramic integration test failed because of that.

**Comment by ukstv** [comment]
@stbrody all green now.

**Review by stbrody — APPROVED** [review]
lgtm

**Pull request merged**